### PR TITLE
Drone v1 docs

### PR DESCRIPTION
**Pull request by nefischer**

### CODE REVIEW

File: docs/how-to-docs/drone-ci.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+- No long-running agents: As with above, there is no longer a requirement for long running agents to run Drone pipeline steps within. This helps keep costs down significantly as the CI environment can run at a lower capacity by default and scale automatically when required.
+- Cron Scheduling: Drone CI now supports the use of [cron jobs](https://docs.drone.io/cron/) to execute pipelines on time-based schedules.
+- Parallelism: Multiple Pipeline objects can be defined for a single repository and executed in parallel. In addition, build steps can be described as a [directed acyclic graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph), allowing for more complex execution flows.
+- Organisation Secrets: Secrets can be created and managed at an Organisation level, i.e. made available to all repositories within your Gitlab group.
+- Container Registry Authentication: Pipeline steps referencing images stored in ACP ECR or Artifactory can be pulled by default, without the need to manually authenticate with separate credentials.
+
```
Comment (KashifSaadat) [inline docs/how-to-docs/drone-ci.md:29]: Add a note here for Starlark and any other recent changes or plugins that were introduced?

File: docs/how-to-docs/drone-ci.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+### Repository Migration
+
+An initial database migration has occurred from 01/07/2020 to populate the new Drone CI instances with repository secrets, in order to ease the migration process for users. There are required changes to your `.drone.yml` file however, before you are able to activate and execute any new builds.
+
+1. Download and install the latest version of the [drone-cli](https://docs.drone.io/cli/install/) binary.
+
+    It is likely that you will have to interact with the old and the new instances of drone. If that's the case, you might want to rename the `drone` binary you've just downloaed to `drone1` so that you can use the `drone` CLI to interact with the old drone instances (v0.8) and `drone1` to interact with the new ones (v1). Please note that any further reference to `drone` as the CLI refers to the latest version of drone-cli for the v1 instances.
```
Comment (KashifSaadat) [inline docs/how-to-docs/drone-ci.md:36]: `downloaed` => `downloaded`

File: docs/how-to-docs/drone-ci.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+1. Use the drone-cli to initially convert to the new format: `drone convert` (run `drone convert --save` after validating the output).
+1. For each pipeline definition, specify the `type` to state using the kubernetes build runner. For example:
+
+    ```yml
+    kind: pipeline
+    name: default
+    type: kubernetes
+    ```
+
+1. If your pipeline deal with `deployment` events, replace that event type with `promote`. For example:
```
Comment (KashifSaadat) [inline docs/how-to-docs/drone-ci.md:47]: `deal` => `deals`

File: docs/how-to-docs/drone-ci.md (hunk 1 above)
Comment (nefischer) [reply docs/how-to-docs/drone-ci.md:30]: Note on starlark added

File: docs/how-to-docs/drone-ci.md (hunk 2 above)
Comment (nefischer) [reply docs/how-to-docs/drone-ci.md]: fixed

File: docs/how-to-docs/drone-ci.md (hunk 3 above)
Comment (nefischer) [reply docs/how-to-docs/drone-ci.md]: fixed

**Review by KashifSaadat — APPROVED** [review]

**Pull request merged**